ampersand-select-view.js

```diff
@@ -76,6 +76,8 @@ module.exports = View.extend({
         this.startingValue = opts.value;
         this.yieldModel = (opts.yieldModel === false) ? false : true;
 
+        this.tabindex = opts.tabindex || '0';
+
```

Comment:
Help for other reviewers: this is one of the two actual code additions that isn't whitespace cleanup.